src/Microsoft.ML.HalLearners/OlsLinearRegression.cs

```diff
@@ -604,6 +604,18 @@ private static VersionInfo GetVersionInfo()
         /// </summary>
         public IReadOnlyCollection<Double> PValues => _pValues.AsReadOnly();
 
+        /// <summary>
+        /// Constructs a new OLS regression model parameters from trained model.
+        /// </summary>
+        /// <param name="env">The Host environment.</param>
+        /// <param name="weights">The weights for the linear predictor. Note that this
```

Comment:
This is not very clear. It'd be better to explain each element in the array; for example, `the i-th element in weights, denoted by weights[i], the the weight of the i-th feature.` Please revise other vector/matrix attributes. Without this information, user may not be able to extract, for example, bias's standard deviation from standardErrors.